packages/near-membrane-base/src/membrane.ts

```diff
@@ -455,64 +454,51 @@ export function createMembraneMarshall() {
 
         function getTargetTraits(target: object): TargetTraits {
             let targetTraits = TargetTraits.None;
+            let targetIsArray = false;
+            try {
+                targetIsArray = isArrayOrNotOrThrowForRevoked(target);
+            } catch {
+                targetTraits |= TargetTraits.Revoked;
```

Comment:
should we return right after?

Reply:
nevermind, I think you still want to know the type of the revoked... all good!

Reply:
Right. We can still get type info if it's revoked.